app/models/batch_process.rb

```diff
@@ -272,6 +289,14 @@ def create_new_parent_csv
   # rubocop:enable  Metrics/CyclomaticComplexity
   # rubocop:enable Metrics/BlockLength
 
+  def valid_viewing_directions
+    ['left-to-right', 'right-to-left', 'top-to-bottom', 'bottom-to-top', nil]
+  end
+
+  def valid_display_layouts
```

Comment:
I think you can replace this method with something like `parent_object.viewing_hints.include?` https://github.com/yalelibrary/yul-dc-management/blob/main/app/models/parent_object.rb#L78